plugin/trino-hive/src/main/java/io/trino/plugin/hive/ViewReaderUtil.java

```diff
@@ -183,7 +183,7 @@ public static boolean isTrinoMaterializedView(String tableType, Map<String, Stri
         // https://github.com/trinodb/trino/blame/ff4a1e31fb9cb49f1b960abfc16ad469e7126a64/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java#L898
         return tableType.equals(VIRTUAL_VIEW.name()) &&
                 "true".equals(tableParameters.get(PRESTO_VIEW_FLAG)) &&
-                tableParameters.get(TABLE_COMMENT).equalsIgnoreCase(ICEBERG_MATERIALIZED_VIEW_COMMENT);
+                ICEBERG_MATERIALIZED_VIEW_COMMENT.equalsIgnoreCase(tableParameters.get(TABLE_COMMENT));
```

Comment:
Please apply the same handling on https://github.com/trinodb/trino/pull/18800/files/520d9b200b12e54120560da22dcf14b9dd8d7c75#diff-ce243e62b3d6f2ba7a3afb20e69702c55ce47b7579e66391d3a96b42605cb4e3R171